lib/iex/lib/iex/helpers.ex

```diff
@@ -398,6 +398,9 @@ defmodule IEx.Helpers do
   ## Examples
 
       iex> i(1..5)
+      
+    Will print:
```

Comment:
Yes, 4 spaces means it will be rendered with monospaced font so we keep `iex>` and the stuff that is printed at 4 spaces indenation, the rest goes at 0 spaces indentation.